pkg/helm/init.go

```diff
@@ -0,0 +1,115 @@
+package helm
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/mholt/archiver"
+	"github.com/pkg/errors"
+)
+
+// Init inits the Helm server (Tiller) if not running, else, ensures the
+// Helm client matches the installed Tiller version
+func (m *Mixin) Init() error {
+	tillerVersion, err := m.getTillerVersion()
+	if err != nil {
+		fmt.Fprintf(m.Out, "Tiller may not be running; attempting to init.")
+		initCmd := m.NewCommand("helm", "init", "--upgrade", "--wait")
```

Comment:
Is the  `--upgrade` flag necessary since tiller isn't installed?

Can this also have errored out because the kubeconfig wasn't there or wasn't connecting to the cluster properly? If that happens, then it will just fail a second time right with "unable to init tiller: some connection failure message"?

Reply:
Yes, it would just error out again if some more general/underlying issue is present, for example:
```
Tiller may not be running; attempting to setup RBAC and init.
err: failed to setup RBAC for Tiller: unable to create RBAC resource: The connection to the server localhost:52378 was refused - did you specify the right host or port?
: exit status 1
Error: failed to setup RBAC for Tiller: unable to create RBAC resource: The connection to the server localhost:52378 was refused - did you specify the right host or port?
: exit status 1
Error: mixin execution failed: exit status 1
```
(testing RBAC setup additions here; provided a kubeconfig to a non-functioning cluster)

Reply:
I'm hesitant to get into the weeds with parsing stderr and switching/adding logic... rather, just failing at that point.  What do you think?

Reply:
Oh sorry! I wasn't suggesting that you parse the error. I was just making sure that if the problem was something other than "tiller isn't installed" that it would get caught here and they would see a useful error message.

Reply:
No apology necessary :) Good call, perhaps if the error message is other than `could not find a ready tiller pod`, we'll explicitly error out/exit early.

Reply:
Updated; PTAL when convenient!